proto/encore/parser/meta/v1/meta.proto

```diff
@@ -8,56 +8,56 @@ import "encore/parser/schema/v1/schema.proto";
 // where a package may refer to its own objects.
 message QualifiedName {
-  string pkg = 1; // "rel/path/to/pkg"
+  string pkg  = 1; // "rel/path/to/pkg"
   string name = 2; // ObjectName
 }
 
 message Package {
-  string rel_path = 1;                  // import path relative to app root ("." for the app root itself)
-  string name = 2;                      // package name as declared in Go files
-  string doc = 3;                       // associated documentation
-  string service_name = 4;              // service name this package is a part of, if any
-  repeated string secrets = 5;          // secrets required by this package
-  repeated QualifiedName rpc_calls = 6; // RPCs called by the package
-  repeated TraceNode trace_nodes = 7;
+  string                 rel_path     = 1; // import path relative to app root ("." for the app root itself)
+  string                 name         = 2; // package name as declared in Go files
+  string                 doc          = 3; // associated documentation
+  string                 service_name = 4; // service name this package is a part of, if any
+  repeated string        secrets      = 5; // secrets required by this package
+  repeated QualifiedName rpc_calls    = 6; // RPCs called by the package
+  repeated TraceNode     trace_nodes  = 7;
 }
 
 message Service {
-  string name = 1;
-  string rel_path = 2; // import path relative to app root for the root package in the service
-  repeated RPC rpcs = 3;
+  string               name       = 1;
+  string               rel_path   = 2; // import path relative to app root for the root package in the service
+  repeated RPC         rpcs       = 3;
   repeated DBMigration migrations = 4;
 }
 
 message DBMigration {
-  string filename = 1;    // filename
-  int32 number = 2;       // migration number
+  string filename    = 1; // filename
+  int32  number      = 2; // migration number
   string description = 3; // descriptive name
 }
 
 message RPC {
-  string name = 1;                    // name of the RPC endpoint
-  string doc = 2;                     // associated documentation
-  string service_name = 3;            // the service the RPC belongs to.
-  AccessType access_type = 4;         // how can the RPC be accessed?
-  schema.v1.Decl request_schema = 5;  // request schema, or nil
-  schema.v1.Decl response_schema = 6; // response schema, or nil
-  Protocol proto = 7;
-  schema.v1.Loc loc = 8;
-  Path path = 9;
-  repeated string http_methods = 10;
+  string                    name            = 1; // name of the RPC endpoint
+  string                    doc             = 2; // associated documentation
+  string                    service_name    = 3; // the service the RPC belongs to.
+  AccessType                access_type     = 4; // how can the RPC be accessed?
+  optional schema.v1.Type  request_schema  = 5; // request schema, or nil
+  optional schema.v1.Type  response_schema = 6; // response schema, or nil
```

Comment:
The optionals here are for the `ts-proto` generator to add the `| undefined` suffixes to these types.

By default that generator either makes every sub-message type optional, or none of them. The optional keyword allows us to force the type to be optional, while keeping the others non-optional in the generated types